rr_computer_vision/scripts/Lane-Detection-Test-Single-Image.ipynb

```diff
@@ -2,24 +2,24 @@
  "cells": [
```

Comment:
Are the changes you made in here the same as the other jupyter notebook file you added? If so and there's no value in having them in here should probably get rid of it